tests/utils/ado/ado.sh

```diff
@@ -9,87 +9,52 @@ group="${args[0]}"
 
 command -v python
 python -V
-
 if [ "$2" = "2.7" ]
 then
-    command -v pip
-    pip --version
-    pip list --disable-pip-version-check
+    echo "The specified environment is Python2.7"
 else
-    if [ "$2" = "3.8" ]
-    then
-        sudo apt update
-        sudo apt install software-properties-common
-        sudo add-apt-repository ppa:deadsnakes/ppa
-        sudo apt install python"$2" -y
-        sudo update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.8 1
-    fi
-
-    command -v pip3
-    pip3 --version
-    pip3 list --disable-pip-version-check
+    alias pip='pip3'
+    sudo apt update
```

Comment:
could you check if `"$UID" -eq "0"` here instead of always performing sudo? It would be nice to be able to run this easily from inside a Docker container